docs/install-gitlab.md

```diff
@@ -141,6 +138,10 @@ frogbot-scan:
     # For Linux / MacOS runner:
     - curl -fLg "https://releases.jfrog.io/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
     - ./frogbot ${FROGBOT_CMD}
+      
+    # For Linux / MacOS runner using Artifactory remote repository:
+    # - $JF_URL/artifactory/$JF_RELEASES_REPO/artifactory/frogbot/v2/[RELEASE]/getFrogbot.sh" | sh
+    # - ./frogbot ${FROGBOT_CMD}
```

Comment:
Shouldn't we implement the "if statement" here as well?
Can we also support Windows here?

Reply:
done